Paola changes to Release Notes (#1785)
astronomer/docs: astro/release-notes.md

### New Astro Cloud IDE cell types

> The Astro Cloud IDE now supports the following cell types, each of which allow you to create a new type of task:

Reviewer: ```suggestion
To simplify and speed the creation of new tasks, the following new cell types are now available in the Astro Cloud IDE::
```

> To simplify and speed the creation of new tasks, the following new cell types are now available in the Astro Cloud IDE:

Reviewer 2: ```suggestion
To simplify the creation of new tasks, the following new cell types are now available in the Astro Cloud IDE:
```

- **SQL**: Run a SQL query against an existing database connection and persist the results of the query in an XCom file for use by other cells. Use this cell type to run smaller queries and store the results in Airflow for quick access in other cells.

> - **Warehouse SQL**: Run a SQL query against an existing database connection and store the results in your data warehouse. Use this cell type for data operations that require more storage and a higher level of reliability.

Reviewer: ```suggestion
- **Warehouse SQL**: Run a SQL query against an existing database connection and store the query results in your data warehouse. Use this cell type for data operations that require more storage and reliability.
```

> - **Markdown**: Add in-line comments in Markdown to your generated DAG code. Use this cell type to document steps in your pipeline and make it easier for members of your team to collaborate on the same pipeline.

Reviewer: ```suggestion
- **Markdown**: Add inline Markdown comments to your generated DAG code. Use this cell type to document code decisions and to make it easier for team members to collaborate on shared pipeline.
```
![image](https://user-images.githubusercontent.com/104205257/214390367-5a113e40-0fa6-4cd1-aab7-888d9184246f.png)

> Previously, you could only create Python cell types in the Cloud IDE. To learn more, see [Run SQL in the Astro Cloud IDE](cloud-ide/run-sql.md), [Run Python in the Cloud IDE](cloud-ide/run-python.md), and [Add documentation to an Astro Cloud IDE pipeline](cloud-ide/document-pipeline.md).

Reviewer: ```suggestion
For more information about a specific cell type, see [Run SQL in the Astro Cloud IDE](cloud-ide/run-sql.md), [Run Python in the Cloud IDE](cloud-ide/run-python.md), and [Add documentation to an Astro Cloud IDE pipeline](cloud-ide/document-pipeline.md).
```

### Bug fixes 

> - After you click **Clear** or **Mark Success** for a task instance in the Airflow UI, you are now correctly redirected to the **Grid** view of the corresponding DAG. Previously, you were redirected to the **DAGs** home page.

Reviewer: ```suggestion
- After clicking **Clear** or **Mark Success** for a task instance in the Airflow UI, you are now redirected to the **Grid** view of the corresponding DAG instead of the **DAGs** home page.
```
